### PR TITLE
Updated supported  clients table

### PR DESCRIPTION
**Pull request by Loquacity**
https://bugzilla.suse.com/show_bug.cgi?id=1087841
https://github.com/SUSE/spacewalk/issues/6601

### CODE REVIEW

**Comment by Loquacity** [comment]
@jcayouette I have memories of discussing updating this  table with green checks and red crosses instead of the "supported"/"unsupported" statements. Am I imagining this?

**Comment by keichwa** [comment]
What about using plain unicode sign?  They are not neccessarily green and red, but there is something to choose from (we'll have to explain what they mean):

https://en.wikipedia.org/wiki/Check_mark
U+2713 ✓ CHECK MARK
U+2705 ✅ WHITE HEAVY CHECK MARK

https://en.wikipedia.org/wiki/X_mark
✗ | U+2717 | BALLOT X (cross)

❌ | U+274C | CROSS MARK
❎ | U+274E | NEGATIVE SQUARED CROSS MARK

**Comment by Loquacity** [comment]
> What about using plain unicode sign? They are not neccessarily green and red, but there is something to choose from (we'll have to explain what they mean):

Ah, yes, this would work!

> U+2713 ✓ CHECK MARK
> ✗ | U+2717 | BALLOT X (cross)

I like these ones :)

File: modules/installation/pages/client-requirements.adoc [hunk 1]
```diff
@@ -7,19 +7,23 @@
...
-|_Latest minor release {rhnminrelease7}_               | {x86}_64                                     | Supported           | Supported
-| Open Enterprise Server 2015, 2015 SP1, 2018          | {x86}_64                                     | Supported           | Supported
-| {opensuse} Leap 15.1                                 | {x86}_64                                     | Unsupported         | Supported
-| {ubuntu} 16.04                                       | {x86}_64                                     | Unsupported         | Supported
-| {ubuntu} 18.04                                       | {x86}_64                                     | Unsupported         | Supported
+| Operating System | Architecture | Traditional Clients | Salt Clients
+| {sle} 15 | {x86}_64, IBM {ppc} ({ppc64le}), {zseries}, ARM | Supported | Supported
+| {sle} 12 | {x86}_64, IBM {ppc} ({ppc64le}), {zseries}, ARM | Supported | Supported
+| {sle} 11 | {x86}, {x86}_64, {ipf}, IBM {ppc}, {zseries} | Supported | Supported
+| {sle} 10 SP3+ | {x86}, {x86}_64, {ipf}, IBM {ppc}, {zseries} | Supported | Supported
```
Comment (mcalmer) [inline modules/installation/pages/client-requirements.adoc]: SLES 10 is out of any support. I am not sure if we want to lists systems which might work. Let's remove this line.

File: modules/installation/pages/client-requirements.adoc [hunk 2]
```diff
@@ -7,19 +7,23 @@
...
+| {rhel} 8 | {x86}_64 | TBD | TBD
```
Comment (mcalmer) [inline modules/installation/pages/client-requirements.adoc]: As of now RHEL 8 is not supported. Either we remove this line for now, or we write something like future or comming soon.

File: modules/installation/pages/client-requirements.adoc [hunk 3]
```diff
@@ -7,19 +7,23 @@
...
+| {rhel} 7 | {x86}_64 | Supported | Supported
+| {rhel} 6 | {x86}, {x86}_64 | Supported | Supported
+| CentOS 7 | {x86}, {x86}_64 | TBD | TBD
+| CentOS 6 | {x86}, {x86}_64 | TBD | TBD
+| {opensuse} Leap 15 | {x86}_64 | TBD | TBD
+| {ubuntu} 16.04 | {x86}_64 | Not supported | Supported
+| {ubuntu} 18.04 | {x86}_64 | Not supported | Supported
+| Microsoft Windows 10 | {x86}_64 | TBD | TBD
```
Comment (mcalmer) [inline modules/installation/pages/client-requirements.adoc]: No, we do not support windows

File: modules/installation/pages/client-requirements.adoc [hunk 4]
```diff
@@ -7,19 +7,23 @@
...
+| {sle} 15 | {x86}_64, IBM {ppc} ({ppc64le}), {zseries}, ARM | Supported | Supported
+| {sle} 12 | {x86}_64, IBM {ppc} ({ppc64le}), {zseries}, ARM | Supported | Supported
+| {sle} 11 | {x86}, {x86}_64, {ipf}, IBM {ppc}, {zseries} | Supported | Supported
+| {sle} 10 SP3+ | {x86}, {x86}_64, {ipf}, IBM {ppc}, {zseries} | Supported | Supported
+| {rhel} 8 | {x86}_64 | TBD | TBD
+| {rhel} 7 | {x86}_64 | Supported | Supported
+| {rhel} 6 | {x86}, {x86}_64 | Supported | Supported
+| CentOS 7 | {x86}, {x86}_64 | TBD | TBD
+| CentOS 6 | {x86}, {x86}_64 | TBD | TBD
+| {opensuse} Leap 15 | {x86}_64 | TBD | TBD
```
Comment (mcalmer) [inline modules/installation/pages/client-requirements.adoc]: Please change to 15.1 and set salt to supported while traditional is unsupported

**Comment by jcayouette** [comment]
> What about using plain unicode sign? They are not neccessarily green and red, but there is something to choose from (we'll have to explain what they mean):
> 
> https://en.wikipedia.org/wiki/Check_mark
> U+2713 ✓ CHECK MARK
> U+2705  WHITE HEAVY CHECK MARK
> 
> https://en.wikipedia.org/wiki/X_mark
> ✗ | U+2717 | BALLOT X (cross)
> 
>  | U+274C | CROSS MARK
>  | U+274E | NEGATIVE SQUARED CROSS MARK

This is one approach that work just fine, however with asciidoctor we are also able to use font awesome icons, perhaps these are a little more modern. For now however we can stick with this idea, I suggest after release we move to the green and red font awesome icons.

File: modules/installation/pages/client-requirements.adoc (hunk 3 above)
Comment (jcayouette) [reply modules/installation/pages/client-requirements.adoc]: @mcalmer We do not support it, but can't windows clients be connected via Salt or am I dreaming? :grin:

File: modules/installation/pages/client-requirements.adoc (hunk 2 above)
Comment (Loquacity) [reply modules/installation/pages/client-requirements.adoc]: I've got a note   about what  TBD means now: ``Fields marked as `TBD` are under consideration, and may or may not be supported at a later date.``

Does this resolve the problem?

File: modules/installation/pages/client-requirements.adoc (hunk 3 above)
Comment (Loquacity) [reply modules/installation/pages/client-requirements.adoc]: They're on the list on  the wiki ...

**Comment by Loquacity** [comment]
Hrm, the unicode isn't showing up when i build locally. Am I doing something wrong?

File: modules/installation/pages/client-requirements.adoc (hunk 3 above)
Comment (mcalmer) [reply modules/installation/pages/client-requirements.adoc]: What is this list about? What salt can do or what we support in SUSE Manager ?
AFAIK the list in the WIKI comes from Don(Sales). Maybe as a Sales person he present what could be possible.

File: modules/installation/pages/client-requirements.adoc (hunk 2 above)
Comment (mcalmer) [reply modules/installation/pages/client-requirements.adoc]: Is this also clear for the Customer? For me `TBD` is a placeholder for developers and the decission should be done before it reach the customer.

Maybe we should write `under consideration` ?

But if this is a typical term you find also in other docs I am fine with it.

File: modules/installation/pages/client-requirements.adoc (hunk 3 above)
Comment (mcalmer) [reply modules/installation/pages/client-requirements.adoc]: In SUMA 4.0, Windows is definitve not `under consideration` :-)

File: modules/installation/pages/client-requirements.adoc [hunk 5]
```diff
@@ -6,20 +6,26 @@
...
-| {opensuse} Leap 15.1                                 | {x86}_64                                     | Unsupported         | Supported
-| {ubuntu} 16.04                                       | {x86}_64                                     | Unsupported         | Supported
-| {ubuntu} 18.04                                       | {x86}_64                                     | Unsupported         | Supported
+| Operating System | Architecture | Traditional Clients | Salt Clients
+| {sle} 15 | {x86}_64, {ppc}, {ibmz}, ARM | &#2713; | &#2713;
+| {sle} 12 | {x86}_64, {ppc}, {ibmz}, ARM | &#2713; | &#2713;
+| {sle} 11 | {x86}, {x86}_64, {ipf}, IBM {ppc}, {ibmz} | &#2713; | &#2713;
+| {rhel} 8 | {x86}_64 | TBD | TBD
+| {rhel} 7 | {x86}_64 | &#2713; | &#2713;
+| {rhel} 6 | {x86}, {x86}_64 | &#2713; | &#2713;
```
Comment (mcalmer) [inline modules/installation/pages/client-requirements.adoc]: Do we need to name "Expanded Support" extra? 

SLES-ES 6 / 7 is supported like RHEL.

File: modules/installation/pages/client-requirements.adoc (hunk 3 above)
Comment (Loquacity) [reply modules/installation/pages/client-requirements.adoc]: lol, ok taking it out!

File: modules/installation/pages/client-requirements.adoc (hunk 5 above)
Comment (Loquacity) [reply modules/installation/pages/client-requirements.adoc]: Added.

File: modules/installation/pages/client-requirements.adoc (hunk 2 above)
Comment (Loquacity) [reply modules/installation/pages/client-requirements.adoc]: TBD and TBC are both used fairly frequently in tech docs (in my experience, anyway), and we are explaining the term. I think  'Under consideration' is too long for the table. Alternatively, we could put a unicode question mark or something in, to match the style of the ticks and crosses?

Comment (mcalmer) [reply modules/installation/pages/client-requirements.adoc]: Ok, you are the experts. So I am fine with TBD or TBC.

File: modules/installation/pages/client-requirements.adoc [hunk 6]
```diff
@@ -6,20 +6,27 @@
 
 Supported operating systems for traditional and Salt clients are listed in this table.
 
+In this table, &#2713; indicates that clients running the operating system are supported by {scc}, and &#2717; indicates that it is not supported.
```
Comment (mcalmer) [inline modules/installation/pages/client-requirements.adoc]: Please check if {scc} is correct here. I do not now what the replacement is, but "supported by SUSE Customer Center" does not sound right.

A second thing to check are the unicode characters, Here in github they show something different than wanted.

Comment (Loquacity) [reply modules/installation/pages/client-requirements.adoc]: Updated to "supported by {suse}".

**Comment by Loquacity** [comment]
> Hrm, the unicode isn't showing up when I build locally. Am I doing something wrong?

@jcayouette Can you look into this please?

File: modules/installation/pages/client-requirements.adoc (hunk 6 above)
Comment (mcalmer) [reply modules/installation/pages/client-requirements.adoc]: Thanks

File: modules/installation/pages/client-requirements.adoc [hunk 7]
```diff
@@ -6,20 +6,27 @@
...
-|_Latest minor release {rhnminrelease7}_               | {x86}_64                                     | Supported           | Supported
-| Open Enterprise Server 2015, 2015 SP1, 2018          | {x86}_64                                     | Supported           | Supported
-| {opensuse} Leap 15.1                                 | {x86}_64                                     | Unsupported         | Supported
-| {ubuntu} 16.04                                       | {x86}_64                                     | Unsupported         | Supported
-| {ubuntu} 18.04                                       | {x86}_64                                     | Unsupported         | Supported
+| Operating System | Architecture | Traditional Clients | Salt Clients
+| {sle} 15 | {x86}_64, {ppc}, {ibmz}, ARM | &#2713; | &#2713;
+| {sle} 12 | {x86}_64, {ppc}, {ibmz}, ARM | &#2713; | &#2713;
+| {sle} 11 | {x86}, {x86}_64, {ipf}, IBM {ppc}, {ibmz} | &#2713; | &#2713;
+| {sls}-ES 7 | {x86}, {x86}_64 | &#2713; | &#2713;
```
Comment (mcalmer) [inline modules/installation/pages/client-requirements.adoc]: We only support 64 bit with expanded support. Please remove {x86} from here.

**Comment by jcayouette** [comment]
Ill have a look

**Comment by Loquacity** [comment]
> Ill have a look

@jcayouette I've updated to the font awesome icons, but they're still not showing up for me on a local build. I need your expertise, please.

**Pull request merged**